lib/Zonemaster/CLI.pm

```diff
@@ -601,6 +603,8 @@ sub add_fake_delegation {
             exit( 1 );
         }
 
+        $name =~ s/\.$//;
```

Comment:
Can the root be a NS in the context of fake delegation? I think it can, according to code I see in Engine ([here](https://github.com/zonemaster/zonemaster-engine/blob/master/lib/Zonemaster/Engine.pm#L112-L114)). @matsduf can you confirm?
Thus it would require `unless $name eq '.';` here as well.

Reply:
Yes, an NS can point at a name server name ".".